docs/dev-proc/versioning.md

```diff
@@ -4,24 +4,27 @@ Dasharo Releases are versioned using [Semantic Versioning](https://semver.org/)
 and [Keep A Changelog](https://keepachangelog.com/en/1.0.0/) to document
 changes introduced in new releases.
 
-Major version zero (0.y.z) is for initial development and may not support all
-Dasharo Quality Criteria.
+Major version zero (0.y.z) is for initial development or first release issued
+and may not support all Dasharo Quality Criteria.
 
 The only way to map Dasharo Version to version of Open Source Firmware
 framework or other components included in Dasharo Release is through release
 notes. Link to Dasharo Release Notes for your hardware platform can be found
 in menu on the left side (Supported Hardware->Hardware Model->Releases).
 
-## Dasharo Supporters releases
+## Dasharo Entry Subscription releases
 
-Dasharo Supporters receive firmware updates more frequently than the community.
-Number of updates per year depends on the number of Dasharo Subscriptions sold
-and the availability of other funding (e.g., NLNet, corporate sponsors,
-community donations) but is less than 2 updates per year. Dasharo Supporters
-updates are characterized by a changing patch version (`z`). Fixes and features
-introduced in Dasharo Supporters releases will also be available later in
-community releases. In short, being a Dasharo Supporter gives an early access to
-the newest features and fixes.
+Dasharo Supporters receive firmware updates more frequently than the
+community. Number of updates per year depends on the number of Dasharo
+Subscriptions sold and the availability of other funding (e.g., NLNet,
+corporate sponsors, community donations) but is less than 2 updates per year.
```

Comment:
This sounds weird. Dasharo Community Releases I agree, there would be less than two. But we should have no such arbitrary limit for Dasharo Entry Subscription Releases (previously Dasharo Supporters Releases).

Reply:
I haven't changed the paragraph except the nomenclature (but it got rewrapped and GH detected it as an edition of whole paragraph). I think it was meant to be `no less than 2 updates per year` for subscribers @pietrushnic

Reply:
@miczyg1, please fix it, since we have occasion to get rid of such minor typos.

Reply:
@pietrushnic I would actually get rid of that part. How can we guarantee there is enough subscription bought to provide two releases per year? And how long can we guarantee such schedule? Sounds like TOS

Reply:
@miczyg1 you are right, but we should explain difference between Dasharo Community Release and Dasharo Entry Subscribers Release.

Reply:
The first sentence already says:

Dasharo ~~Supporters~~ Entry Subscribers receive firmware updates more frequently than the community

So I think this change should suffice @pietrushnic :

```suggestion
corporate sponsors, community donations).
```

Reply:
Then we also have:

```
Fixes and features introduced in Dasharo Entry Subscription
releases will also be available later as community releases with public
pre-built binaries in the respective release pages. In short, being a Dasharo
Supporter gives an early access to the newest features and fixes.
```

Reply:
Terminology is not consistent. We should have finished a number of terms, and now we have DES, community release, and Dasharo Supporter. Some are written with big letters like legal/marketing names. Some are small, and I have no idea how to associate those.

During DUGs, so for the last five months, we have tried to use just two terms regarding releases:
- Dasharo Community Release
- Dasharo Entry Subscription Release (previous Dasharo Supporters Release for Dasharo Support Entrance Subscribers)

According to the above, we may have:
- Dasharo Community Release subscribers - those who subscribed to free of charge mailing list on docs.dashro.com
- Dasharo Entry Subscription Release subscribers (:man_facepalming:) - those who paid for DES or received it because of other reasons

The language used in this paragraph may mean the same or may mean something different. Consistency in the language is very important if we don't want to have endless support discussions and, in case of questions, send reliable sources of answers.

Reply:
I hope I applied consistent nomenclature everywhere @pietrushnic ccddb79e6f66511cec75e029aebb31490ae56b77